### ISSUE TITLE
iPhone 5s ipfs hash stuck on loading

### ISSUE THREAD

**Issue by kikipluche** [swarmcity/sc-boardwalk#161]

**Comment by sophiii** [comment]
Can you confirm this issue still exists?

**Comment by sophiii** [comment]
can you test this with chrome on iphone?

**Comment by kikipluche** [comment]
@sophiii  yes, on chrome on iphone I cannot type  my pasword, it is stuck there, cursor blinking...

**Comment by sponnet** [comment]
@kikipluche - what is the current problem with iPhone ? Can you confirm is this is still an issue , or if has been solved ? Thanks. I don't have an IPhone to test on.

**Comment by kikipluche** [comment]
@sponnet we are still receiving errors on iphone. I myself am not able to restore from ipfs, gets stuck on the password view...

**Comment by bkawk** [comment]
On this line https://github.com/swarmcity/sc-boardwalk/blob/master/polymer.json#L34 try changing it to 
`"js": {"minify": true, "compile": true},` if that doesn't fix it then I suspect the build warnings relating to IPFS need looking into  more
